src/components/FacetsEditor/FacetsEditor.js

```diff
@@ -25,13 +25,13 @@ class FacetsEditor extends Component {
     const { onChange, component, previewDefinition, readOnly } = this.props
 
     return (
-      <Grid className="no-gutters">
```

Comment:
We should consider reintroducing the Grid later on, but let's go with that for now.

Reply:
Sorry. I didn't see your comment earlier.
The `<Grid>` adds a `container` class which in Bootstrap it specifies a fixed width of 1170px (for large screens (min-width: 1200px))

We're using Grid, just not inside the smaller sections like the FacetsEditor